src/common/dns_resolve.cc

```diff
@@ -314,7 +314,7 @@ int DNSResolver::resolve_srv_hosts(CephContext *cct, const string& service_name,
   }
 #endif
   if (len < 0) {
-    lderr(cct) << "res_search() failed" << dendl;
+    lderr(cct) << "res_search() failed" << query_str << dendl;
```

Comment:
@renhwztetecs I meant, `len` is not always getting the return value of `res_search()`. `#ifdef HAVE_RES_NQUERY` then it is getting the return value of `res_nsearch()`. So what do you think about the error message "res_search() failed" in your changes ?